### ISSUE TITLE
AutoComplete: change event not triggered

### ISSUE THREAD

**Issue by DanielPettersson** [primefaces/primefaces#6711]
**Describe the defect**
Autocomplete does no longer trigger the change ajax event. Though in the same setup inputText does trigger change ajax event. Using Primefaces 8.0, autocomplete triggers change ajax event.

Autocomplete still triggers itemSelect if using a completeMethod, and also triggers blur event. It's just change event that is effected. I guess the problem is caused by https://github.com/primefaces/primefaces/commit/6876cfc31ac18229a228f6f61924d9b852fc4224

The defect only occurs when CSP is enabled, by adding:

`<context-param>
	<param-name>primefaces.CSP</param-name>
	<param-value>true</param-value>
</context-param>`

to web.xml

**Environment:**
 - PF Version: _8.0.5_
 - JSF + version : _Jakarta Faces 2.3.14_
 - Affected browsers: _ALL_

**To Reproduce**
1. Change value in inputText and tab, label is updated
2. Change value in autoComplete and tab, label is not updated

**Expected behavior**
Typing text in an autocomplete and focusing other component should trigger a change event. 

**Example XHTML**
```html
<h:form>

	<p:inputText value="#{autocompleteTestBean.value}">
		<p:ajax event="change" update="val"/>
	</p:inputText>

	<p:autoComplete value="#{autocompleteTestBean.value}">
		<p:ajax event="change" update="val"/>
	</p:autoComplete>
	
	<h:outputLabel id="val" value="#{autocompleteTestBean.value}"/>

</h:form>
```

**Example Bean**
```java
@Component
@Scope(value = "view")
public class AutocompleteTestBean extends ControllerBean implements Serializable {

	@Getter
	@Setter
	private String value;
}
```

**Comment by melloware** [comment]
Introduced with https://github.com/primefaces/primefaces/issues/6061

**Comment by melloware** [comment]
I can't reproduce with 8.0 or 8.0.4 or 9.0-SNAPSHOT.  Can you try my reproducer:
[pf-6711.zip](https://github.com/primefaces/primefaces/files/5735873/pf-6711.zip)

Run `mvn clean jetty:run` and navigate to http://localhost:8080/primefaces-test/test.xhtml

**Comment by melloware** [comment]
No response from OP. Closing until reproducible steps are provided.

**Comment by DanielPettersson** [comment]
Tried reproducing the issue with attached reproducer, but no luck. Also tried running reproducer with Tomcat 9.0.37 and Jakarta faces 2.3.14, still no issue. So there must be something else in our setup that is interfering with the autocomplete.

Thanks for your help!

**Comment by DanielPettersson** [comment]
@melloware This issue only occurs when also enabling Primefaces CSP. I updated the description of the issue with my csp configuration.

By doing this I could reproduce the bug using your attached reproducer.

**Comment by melloware** [comment]
OK reopened ticket let me investigate.

**Comment by melloware** [comment]
This was similar to the problem I fixed for InputNumber: https://github.com/primefaces/primefaces/issues/6142

**Comment by melloware** [comment]
@DanielPettersson Can you try it with this in your web.xml and let me know if it works.
```xml
    <context-param>
        <param-name>primefaces.MOVE_SCRIPTS_TO_BOTTOM</param-name>
        <param-value>true</param-value>
    </context-param>
```

**Comment by DanielPettersson** [comment]
@melloware Yes, the workaround with MOVE_SCRIPTS_TO_BOTTOM works. Nice, thanks!

**Comment by melloware** [comment]
Thanks for reporting back this is fixed in 10.0  However you are better off with MOVE_SCRIPTS as it should make your page load speed a little faster or a lot faster depending on how many components on your page!
